src/build/dotnet/steps/net-core.steps.yaml

```diff
@@ -0,0 +1,73 @@
+# Steps to build .NET Core Projects
+parameters:
+  - name: projects
+    default: '**/*.csproj'
+  - name: tests
+    default: '**/*.Tests.csproj'
+  - name: shouldRunTests
+    default: true
+  - name: applyVersioning
+    default: true
+  - name: artifactName
+    default: '$(Build.DefinitionName)'
+
+steps:
+  - ${{ if eq(parameters.applyVersioning, true) }}:
+    - task: UpdateDotNetVersions@12
+      displayName: .NET Versioning
+      inputs:
+        releaseBranchNames: 'master,main'
+        ExcludePreReleasePostfix: false
+        workingDirectory: ''
+        includeCsproj: true
+        includeNuspec: false
+        packageSource: 'internalPublic'
+        excludePaths: ${{parameters.tests}}
+      env:
+        SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+
+  - task: DotNetCoreCLI@2
+    displayName: .NET Restore
+    inputs:
+      command: restore
+      projects: ${{parameters.projects}}
+      vstsFeed: 'Audacia.Public/AudaciaPublic'
+
+  - task: DotNetCoreCLI@2
+    displayName: .NET Build
+    inputs:
+      command: build
+      projects: ${{parameters.projects}}
+      publishWebProjects: false
+      arguments: >
+        --configuration Release
+        --no-restore
+
+  - task: DotNetCoreCLI@2
+    displayName: .NET Test
+    condition: and(succeeded(), eq('${{parameters.shouldRunTests}}', true))
+    inputs:
+      command: test
+      projects: ${{parameters.tests}}
+      arguments: >
+        --collect "Code coverage"
```

Comment:
we've recently removed this as it was leading to OOMs. Not saying we should remove support, but perhaps we should parameterise this in case other projects have the same issue?